src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -4847,6 +5070,47 @@ public new abstract class Arm64 : AdvSimd.Arm64
         ///   LDNF1B Zresult.D, Pg/Z, [Xbase, #0, MUL VL]
         /// </summary>
         public static unsafe Vector<ulong> LoadVectorByteNonFaultingZeroExtendToUInt64(byte* address) => LoadVectorByteNonFaultingZeroExtendToUInt64(address);
+        /// <summary>
+        /// svint16_t svldff1ub_s16(svbool_t pg, const uint8_t *base)
+        ///   LDFF1B Zresult.H, Pg/Z, [Xarray, Xindex]
+        ///   LDFF1B Zresult.H, Pg/Z, [Xbase, XZR]
+        /// </summary>
+        public static unsafe Vector<short> LoadVectorByteZeroExtendFirstFaulting(Vector<short> mask, byte* address) => LoadVectorByteZeroExtendFirstFaulting(mask, address);
```

Comment:
Since we are just generating the `[Xbase, XZR]` version, can you delete the `[Xarray, Xindex]` comment from here and in other places of this file and `Sve.PlatformNotSupportedException.cs`?